src/Tools/Source/RunTests/Program.cs

```diff
@@ -42,7 +44,9 @@ internal static int Main(string[] args)
                 cts.Cancel();
             };
 
-            return Run(options, cts.Token).GetAwaiter().GetResult();
+            var result = Run(options, cts.Token).GetAwaiter().GetResult();
+            CheckTotalDumpFilesSize();
```

Comment:
I'd push back on that. The dumps are only captured in our jobs in two cases:

1. Jobs hang and eventually our timeout is hit.
1. Xunit crashes and collect a dump file to see what happens (this PR).

Both these cases are rare and lacking the full dump files there is literally no way for us to find out what is going wrong here. We don't keep the builds long so I'm not concerned about the size here.